presidio-analyzer/README.MD

```diff
@@ -1,39 +1,153 @@
 # Presidio analyzer
 
 ## Description
-The presidio-analyzer is a Python based service which does the actual detection of PII entities.
+The Presidio analyzer is a Python based service for detecting PII entities in text.
```

Comment:
@omri374 general question, do we still need the presidio-analyzer folder? since we are under the persidio directory, maybe it should be just analyzer? not critical, just an idea...

Reply:
In majority of cases the folder's name is the python package's name. If we call this 'analyzer' then you would also need to call 'pip install analyzer'. There are some ways around this but this is the standard as I know it.

Reply:
lets leave the name as is